sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md

```diff
@@ -325,10 +325,10 @@ class AsyncHttpResponse:
     async def read(self) -> bytes:
         """Read the response's bytes into memory."""
 
-    async def iter_raw(self, **kwargs: Any) -> AsyncIterator[bytes]:
+    def iter_raw(self, **kwargs: Any) -> AsyncStreamable[bytes]:
         """Asynchronously iterates over the response's bytes. Will not decompress in the process."""
 
-    async def iter_bytes(self, **kwargs: Any) -> AsyncIterator[bytes]:
+    def iter_bytes(self, **kwargs: Any) -> AsyncStreamable[bytes]:
```

Comment:
Do further tests on whether this breaking.
Also check against aiohttp etc for whether this _should_ be async method.